python-lib/robust_session.py

```diff
@@ -1,6 +1,8 @@
 import time
+import copy
```

Comment:
🧐 question: is it used ?

Reply:
👍 Done in [b21dd28](https://github.com/dataiku/dss-plugin-sharepoint-online/pull/53/commits/b21dd28692865e6cd3a8d56c66849ffa347e8d65)